src/lib/y2storage/clients/inst_disk_proposal.rb

```diff
@@ -99,7 +98,16 @@ def guided_setup(settings)
       end
 
       def expert_partitioner
-        Y2Partitioner::Clients::Main.run
+        dialog = Y2Partitioner::Dialogs::Main.new
+        result = dialog.run(storage_manager.y2storage_probed, @devicegraph)
+        case result
+        # TODO: method Y2Partitioner::Dialogs::Main#run does not return :abort when aborting.
```

Comment:
Well, there is nothing TODO here. We are just prepared to the moment in which the dialog is fixed. This is, at most, a `NOTE`, but not a `TODO`.